src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  display: inline-block;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 40px;
+  height: 40px;
+  background-image: url(images/logo.png);
+}
+
+.navBar {
+  display: inline-block;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  text-transform: uppercase;
+  font-size: 12px;
+  align-items: baseline;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.eachButton {
```

Comment:
the name is not really good because actually we have links but not buttons (link opens some page, button just does something)

and `each` does not help to understand the why it is different from just `button`